project/BuildSettings.scala

```diff
@@ -14,7 +14,6 @@ object BuildSettings {
     organization := "org.lichess",
     scalaVersion := globalScalaVersion,
     resolvers ++= Dependencies.Resolvers.commons,
-    parallelExecution in Test := false,
```

Comment:
Is it?  I'm removing the directive that turns off parallel testing. Sbt runs tests in parallel by default